src/export/StructureDefinitionExporter.ts

```diff
@@ -30,6 +31,30 @@ export class StructureDefinitionExporter {
     if (fshDefinition.description) structDef.description = fshDefinition.description;
   }
 
+  /**
+   * Sets the rules for the StructureDefinition
+   * @param {StructureDefinition} structDef - The StructureDefinition to set rules on
+   * @param {Profile | Extension} fshDefinition - The Profile or Extension we are exporting
+   */
+  private setRules(structDef: StructureDefinition, fshDefinition: Profile | Extension) {
```

Comment:
I may be guilty of forgetting to do this myself, but I think it's good practice to always declare the intended return type of the function (which, in this case, would be `void`).  While ts can infer the return type, declaring it is more intentional and ensures that your function body adheres to that intent.